pallets/liquidity-mining/Cargo.toml

```
@@ -0,0 +1,65 @@
+[package]
+name = "pallet-liquidity-mining"
+version = "1.3.0"
+description = "Liquidity mining"
+authors = ["GalacticCouncil"]
+edition = "2018"
+homepage = "https://github.com/galacticcouncil/Basilisk-node"
+license = "Apache 2.0"
+repository = "https://github.com/galacticcouncil/Basilisk-node"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { default-features = false, features = ["derive"], package = "parity-scale-codec", version = "2.3.1" }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+serde = { features = ["derive"], optional = true, version = "1.0.101" }
+sp-arithmetic = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
+pallet-uniques = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.17", default-features = false }
+
+
+# ORML dependencies
+orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "aac79b3b31953381669a2ffa9b3e9bfe48e87f38", default-features = false }
+
+# HydraDX dependencies
+warehouse_liquidity_mining = { package="pallet-liquidity-mining", git = "https://github.com/galacticcouncil/warehouse", rev="3230d9d1e6eaed6305b1b67587437415cd34e268", default-features = false }
```

nitpick: convention is to use `-` in crate names in toml files
```suggestion
warehouse-liquidity-mining = { package="pallet-liquidity-mining", git = "https://github.com/galacticcouncil/warehouse", rev="3230d9d1e6eaed6305b1b67587437415cd34e268", default-features = false }
```